aws_s3_rules/aws_s3_access_ip_whitelist.py

```diff
@@ -1,24 +1,30 @@
 from ipaddress import ip_network
-# IP addresses (in CIDR notation) indicating approved IP ranges for accessing S3 buckets
-IP_WHITELIST = {
+BUCKET_NAMES = {
+    # Example bucket names go here
```

Comment:
Should this comment say something like "restricted bucket names go here" or something?

Reply:
Yep, adding